main.tex

```diff
@@ -49,7 +49,7 @@
 \setcounter{chapter}{-2}
 \input{order.tex}
 
-%% Print gls
-\printglossaries
+%% Add glossaries back in when a number of terms have been defined
```

Comment:
Perhaps change this to a \todo{}?

Reply:
I usually keep the todo for citations or content sections that haven't been finished. If we don't include a glossary, it won't look unfinished, just not existent.